src/main.rs

```diff
@@ -152,6 +144,14 @@ impl RpcService {
         }
     }
 
+    fn to_multiplier(&self, difficulty: u64) -> f64 {
+        (self.base_difficulty.wrapping_neg() as f64) / ((difficulty.wrapping_neg())) as f64
```

Comment:
There's 2 unnecessary pairs of parens here: `((difficulty.wrapping_neg()))` . I'd write it like this
```rust
(self.base_difficulty.wrapping_neg() as f64) / (difficulty.wrapping_neg() as f64)
```